packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -25,22 +32,126 @@ impl PostgresDatabase {
     }
 }
 
+impl PostgresDatabase {
+    async fn insert_version_id(&self, id: &Identifier) -> Result<(), DatastoreError> {
```

Comment:
@TimDiekmann how do we feel about documenting functions like this? I think we generally want all pub functions documented, but I don't think we should _never_ document private functions either.

Reply:
I wasn't sure how much value docs would add on these but seeing the changes I actually think this is much better. Thanks :) (although explicitly talking about `version_id` and `base_id` does mean we need to make sure to update these docs when we change how versioning is done in the table schemas